packages/evm/contracts/court/controller/Controller.sol

```diff
@@ -129,6 +134,27 @@ contract Controller is IsContract, Modules, CourtClock, CourtConfig {
         _setModulesGovernor(_governors[2]);
     }
 
+    /**
+    * @dev Fallback function allows to forward calls to a specific address in case it was previously registered
+    *      Note the sender will be always the controller in case it is forwarded
+    */
+    function () external payable {
+        address target = customFunctions[msg.sig];
```

Comment:
Might be worth adding a test for fallback functions; I actually don't know how they would get registered.

Reply:
Addressed in 3e5d18a6eeb5c9e20f57eacea79ba35151820a98